src/dataTypes.ts

```diff
@@ -28,7 +28,7 @@ export interface MarkerFileData {
 export interface Force {
     position: Point3D;   //px, py, pz
     components: Point3D; //vx, vy, vz
-    torque: number;
+    torque: Point3D;
```

Comment:
If we change this, just be aware you'll need to change line 72 which treats torque as a number in `src/views/SelectionInfoView.tsx` in PR #193, otherwise it'll break